### PR TITLE
Ledger derivation path added in constructor

### PR DESCRIPTION
**Pull request by clementbalestrat**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "format": "prettier --write '{tools,lib,src,__tests__}/**/*.js'",
     "abitojs": "node ./tools/abitojs.js",
     "webpack": "webpack --mode=production",
-    "build": "npm run clear && npm run abitojs && npm run format && npm run docs && npm run webpack && npm run test"
+    "build": "npm run clear && npm run abitojs && npm run format && npm run docs && npm run webpack"
```
Comment (justinjmoses) [inline package.json]: Is this intentional?

File: lib/signers/ledgerSigner.js [hunk 2]
```diff
@@ -1,10 +1,11 @@
 import Transport from '@ledgerhq/hw-transport-u2f/lib/TransportU2F';
 import Eth from '@ledgerhq/hw-app-eth/lib/Eth';
 import { getDefaultProvider, utils, Signer } from 'ethers';
+
 const LEDGER_DERIVATION_PATH = "44'/60'/0'/";
 
 class LedgerSigner extends Signer {
-  constructor(provider, chainId, addressIndex) {
+  constructor(provider, chainId, addressIndex, derivationPath) {
```
Comment (justinjmoses) [inline lib/signers/ledgerSigner.js]: Yeah I'd just go with named props:

`constructor({ provider, chainId, addressIndex, derivationPath })` - after all, you're invoking `super` with no args.

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "synthetix-js",
-  "version": "2.10.5",
+  "version": "2.10.5-1",
```
Comment (justinjmoses) [inline package.json]: typically I'd do the version bump in `master` after the PR, that way it's tagged in git correctly, even if you squash-merge these commits into one

Comment (clementbalestrat) [reply package.json]: Yeah good point!

**Pull request merged**